contrib/init/sysvinit-debian/docker

```diff
@@ -131,7 +131,7 @@ case "$1" in
 	restart)
 		check_init
 		fail_unless_root
-		docker_pid=$(cat "$DOCKER_SSD_PIDFILE" 2> /dev/null)
+		docker_pid=$(cat "$DOCKER_SSD_PIDFILE" 2> /dev/null ||:)
```

Comment:
This is currently failing CI due to a formatting issue.  Please reformat the shell scripts with `shfmt -w -bn -ci -sr`.

```suggestion
		docker_pid=$(cat "$DOCKER_SSD_PIDFILE" 2> /dev/null || :)
```

Reply:
done, thank you for your feedback.